### PR TITLE
fix ability to use .. as an infix operator

### PR DESCRIPTION
**Pull request by stevengj**
As noted by @tkelman, the ability to define `..` as an infix operator was accidentally removed in #17623.  This restores that capability, and adds a test.

### CODE REVIEW

**Comment by stevengj** [comment]
@tkelman, I'm guessing the `cmdlineargs` Travis failure on OSX is unrelated?

**Comment by tkelman** [comment]
Yeah, I believe that's what https://github.com/JuliaLang/julia/pull/19590 is WIP trying to fix.

File: test/parse.jl [hunk 1]
```diff
@@ -863,3 +863,6 @@ end
 
 @test QualifiedStringMacro.SubModule.x"" === 1
 @test QualifiedStringMacro.SubModule.y`` === 2
+
+..(x,y) = x + y
```
Comment (JeffBezanson) [inline test/parse.jl]: I would use `let ..(x,y) = x+y ...`, or just test the result of `parse` rather than execute the definition.

Comment (stevengj) [inline test/parse.jl]: ok

**Pull request merged**